src/app/components/forms/post/_community-pill/selector/scroll-helper/scroll-helper.ts

```diff
@@ -0,0 +1,26 @@
+import Vue, { CreateElement } from 'vue';
+import { Component, Prop, Watch } from 'vue-property-decorator';
+import { findRequiredVueParent, propOptional } from '../../../../../../../utils/vue';
+import AppScrollScrollerTS from '../../../../../../../_common/scroll/scroller/scroller';
+import AppScrollScroller from '../../../../../../../_common/scroll/scroller/scroller.vue';
+
+@Component({})
+export default class AppScrollHelper extends Vue {
```

Comment:
We don't export default when they aren't vue files. We do the normal import { ... } syntax to import them into other ocmponents.